xds/internal/balancer/clusterimpl/clusterimpl.go

```diff
@@ -209,11 +197,6 @@ func (b *clusterImplBalancer) updateLoadStore(newConfig *LBConfig) error {
 }
 
 func (b *clusterImplBalancer) UpdateClientConnState(s balancer.ClientConnState) error {
-	if b.closed.HasFired() {
-		b.logger.Warningf("xds: received ClientConnState {%+v} after clusterImplBalancer was closed", s)
-		return nil
-	}
-
 	if b.logger.V(2) {
 		b.logger.Infof("Received update from resolver, balancer config: %s", pretty.ToJSON(s.BalancerConfig))
```

Comment:
While you are here, do you mind fixing this log message:

```golang
  b.logger.Infof("Received configuration: %s", pretty.ToJSON(s.BalancerConfig))
```
At some point in the past, we had some of these LB policies receive configuration from the resolver. But these days most policies receive configs from their parent. So, its better to avoid that confusion and simply say "Received configuration".

Reply:
Done